src/hooks/useCurrentMarketId.ts

```diff
@@ -54,6 +56,13 @@ export const useCurrentMarketId = () => {
     return marketId ?? lastViewedMarket;
   }, [hasMarketIds, marketId]);
```

Comment:
I know its not changed here, but seems like `lastViewedMarket` should be in dep array, maybe `marketIds` also

Reply:
Will update in my follow up!